blockscout-ens/bens-proto/proto/api_config_http.yaml

```
@@ -0,0 +1,18 @@
+type: google.api.Service
+config_version: 3
+
+http:
+  rules:
+    #################### DomainsExtractor ####################
+    - selector: blockscout.bens.v1.DomainsExtractor.SearchAddress
+      post: /api/v1/domains/search-address
+      body: "*"
+
+    - selector: blockscout.bens.v1.DomainsExtractor.GetDetailedDomain
+      post: /api/v1/domains/get-detailed
```

I believe it would be better to make it as get request with more resty paths. E.g:

```suggestion
      get: /api/v1/{chain_id}/detailed-domains/{name}
```

And also name the response structure just as `DetailedDomain` instead of `GetDetailedDomainResponse`

---

domain name can contains emoji and non ascii symbols, so I thought that maybe it is better to put it into the body. surely you can encode emoji into url, we can do it this way if you want